c2corg_ui/templates/helpers/view.html

```diff
@@ -326,12 +326,19 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
 
 ## Float buttons
-<%def name="show_float_buttons(string, doc_id, doc_lang, other_langs, missing_langs)">\
+<%def name="show_float_buttons(doctype, doc_id, doc_lang, other_langs, missing_langs)">\
   <div class="float-buttons" app-view-details>
-    <div class="float-button float-edit" tooltip-placement="left" uib-tooltip="{{'Edit' | translate}}" protected-url-btn url="'${request.route_url( string + '_edit' , id=doc_id, lang=doc_lang)}'">
+    % if doctype != 'outings':
+    <div class="float-button float-edit" tooltip-placement="left" uib-tooltip="{{'Edit' | translate}}" protected-url-btn url="'${request.route_url( doctype + '_edit' , id=doc_id, lang=doc_lang)}'">
       <span class="glyphicon glyphicon-edit"></span>
       <p class="float-button-text" translate>Edit</p>
     </div>
+    % else:
+    <div ng-if="::userCtrl.hasEditRightsOuting(outingUsers)" class="float-button float-edit" tooltip-placement="left" uib-tooltip="{{'Edit' | translate}}" protected-url-btn url="'${request.route_url( doctype + '_edit' , id=doc_id, lang=doc_lang)}'">
```

Comment:
Can't the user permission check be done directly in the ``protected-url-btn`` directive? All checks should be done at the same place.